features/admin.organizations.v1/components/organization-list.scss

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+$organization-active-icon-active-color: #349256;
+$organization-active-icon-inactive-color: #eb4f63;
+
+.organization-active-icon {
+    margin-right: 0.5rem !important;
+    margin-left: 0 !important;
+}
+
+.organization-active-icon-active {
+    color: $organization-active-icon-active-color
+}
+
+.organization-active-icon-inactive {
+    color: $organization-active-icon-inactive-color
+}
```

Comment:
Can we nest the css?

eg: 

.organization-status-icon {
    .active {
     
     }

     .inactive {
     
     }
}

and then we can use `organization-status-icon active` etc

Reply:
ack that will be better will do that

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6640/commits/b34467767ee4bbac6d8f95413ede92091a077118